src/main/java/com/checkmarx/flow/utils/HTMLHelper.java

```diff
@@ -369,6 +370,8 @@ private static void setSCAHtmlBody(ScanResults.XIssue issue, ScanRequest request
     private static void setSASTMDBody(ScanResults.XIssue issue, String branch, String fileUrl,
             FlowProperties flowProperties, StringBuilder body) {
         log.debug("Building MD body for SAST scanner");
+        Map<String, Object> additionalDetails = issue.getAdditionalDetails();
```

Comment:
Please move the declaration closer to its usage.

Reply:
moved